src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -234,10 +234,20 @@ export default class CrossDeviceMobileRouter extends Component<
       return this.setError()
     }
 
+    // NOTE, This is should be removed as soon as motion is supported on Mobile
+    const convertMotionStepToInitialConfig = (steps) => {
+      const initialFaceStep = steps.find((s) => s.type === "activeVideo")?.original;
```

Comment:
How does this even compile? `StepConfig` doesn't have an `original` property.

Reply:
should be fixed now